velox/core/PlanNode.h

```diff
@@ -4811,6 +4860,7 @@ class TopNRowNumberNode : public PlanNode {
 
       return std::make_shared<TopNRowNumberNode>(
           id_.value(),
+          function_.has_value() ? function_.value() : RankFunction::kRowNumber,
```

Comment:
Please use function_.value(), after the new change, this field is always required.

Reply:
Why mark it as resolved?

Reply:
We should use the same format as `partitionKeys`, it is required, so not check if it hasValue. If have check, we should use VELOX_CHECK(function_.has_value()) before access the value

Reply:
Yes